TFitnessFunction.java

```diff
@@ -39,6 +47,7 @@ private int playGame(double[] position) {
 		}
 		System.out.println("For " + generatePositionKey(position) + ": " + s.getRowsCleared());
```

Comment:
sucks that we no longer get to see this line.
was a sanity check for me because it was some indication of the progress of the learning algorithm.